extras/courses.md

```diff
@@ -42,6 +51,8 @@ Courses | Duration | Effort
 
 Courses | Duration | Effort
 :-- | :--: | :--:
+[Data Structures and Algorithms specialization](https://www.coursera.org/specializations/data-structures-algorithms) | 6-8 months | 4-8 hours/weeks
```

Comment:
Someone else pointed this out after you did, but they pointed out there is an edX version: https://www.edx.org/micromasters/ucsandiegox-algorithms-and-data-structures

When choosing between edX and Coursera versions of the same (or essentially the same) course, we prefer to use edX because Coursera uses annoying tricks to get you to pay for things you don't necessarily need to pay for. So can you:
- Replace the Coursera link with the edX link, and
- Change "specialization" to "MicroMasters"?